scripts/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+trap "exit" INT TERM ERR
+trap "kill 0" EXIT
+
+(
+  cd examples/fast-api-server
+  poetry run ./run.sh
```

Comment:
Took a comment to slack about this

Reply:
What about in the Makefile instead?

Reply:
I dont have strong opinions either way. Happy to ship and revisit later if it becomes a problem